models/schemas/reviews.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+const reviewSchema = new mongoose.Schema({
+    title: {
+        type: String,
+        required: true
+    },
+    gameTitle: {
+        type: String,
+        required: true
+    },
+    shortDescription: {
+        type: String,
+        required: true,
+    },
+    review: {
+        type: String,
+        required: true
+    },
+    reviewerName: {
+        type: String,
```

Comment:
Could this be a link to a User object?

Reply:
For sure. I intend to incorporate something like that as my next step. This is just a draft, really. I'm looking at some mongoose stuff on that today.